lib/rules/use-brace-expansion.js

```diff
@@ -27,7 +29,12 @@ module.exports = {
 
     return {
       CallExpression(node) {
-        if (!ember.isComputedProp(node)) return;
+        if (
+          !ember.isComputedProp(node) ||
+          (node.callee.property && ignoredMacros.indexOf(node.callee.property.name) > -1)
+        ) {
+          return;
+        }
```

Comment:
is it just `and` and `or` that should not use this? I guess the conservative route here would be to only let it run for `computed(...)` but not for any of the other computed property macros. are there other macros where brace expansion works and makes sense?

Reply:
> are there other macros where brace expansion works and makes sense

Probably, but IIRC most of them have "special" / "custom" arguments and would cause similar failure  scenarios to `computed.or` / `computed.and`.